packages/expo-updates/ios/Tests/EXUpdatesDatabaseInitializationTests.m

```diff
@@ -590,4 +590,46 @@ - (void)testMigration6To7
   XCTAssertEqual(0, [EXUpdatesDatabaseUtils executeSql:selectDeletedSql3 withArgs:nil onDatabase:migratedDb error:nil].count);
 }
 
+- (void)testMigration7To8 {
+  sqlite3 *db;
+  NSError *initializeError;
+  [EXUpdatesDatabaseInitialization initializeDatabaseWithSchema:EXUpdatesDatabaseV6Schema
```

Comment:
```suggestion
  [EXUpdatesDatabaseInitialization initializeDatabaseWithSchema:EXUpdatesDatabaseV7Schema
```
the way we've been doing this for now has been just copying the old schemas into constants at the top of this file. definitely open to better ideas if you want to change this 🤷‍♂️ but either way would be good to start this test with the correct schema.